login.css

```diff
@@ -0,0 +1,188 @@
+@import url('https://vaspacx-components.netlify.app/style.css');
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600&display=swap');
+
+:root{
+ 
+    --mainBrandColor: #3a7ce3;
```

Comment:
better naming can be done for variables. Generally, the camelCase naming convention is a bad practice in CSS. Try to separate them with '-'.